lib/computer_player.rb

```diff
@@ -0,0 +1,24 @@
+class ComputerPlayer
+  attr_reader :mark
+
+  def initialize(mark, board, display)
+    @board = board
+    @display = display
+    @mark = mark
+  end
+
+  def move
+    @display.make_move_message(@mark)
+    index = retrieve_computer_index(1)
+    @display.computer_thinking
+    sleep 1
```

Comment:
I would put `sleep 1` in a function called `computer_is_thinking` or something similar. Just to articulate what is actually going on there.